include/odp/api/spec/timer_types.h

```diff
@@ -501,12 +501,20 @@ typedef struct odp_timer_periodic_start_t {
 	 */
 	uint64_t freq_multiplier;
 
-	/** Timeout event
+	/** Number of timeout events
 	 *
-	 *  This event is enqueued to the destination queue when the timer expires. The event type
-	 *  must be ODP_EVENT_TIMEOUT.
+	 *  Number of timeout events in the tmo_ev array. This value is set by calling
+	 *  odp_timer_periodic_events().
 	 */
-	odp_event_t tmo_ev;
+	uint32_t num_tmo_ev;
```

Comment:
For application to be able to prepare with large enough pool of timeout events, there could be "max events per periodic timer" capability.

Reply:
Added in v2.